src/components/graphCard/graphCard.js

```diff
@@ -157,7 +164,10 @@ class GraphCard extends React.Component {
                 {cardTitle}
               </Title>
             </CardTitle>
-            <CardActions className={(error && 'blur') || ''}>{children}</CardActions>
+            <CardActions className={(error && 'blur') || ''}>
+              <React.Fragment key="hasActionDisplay">{hasActionDisplay}</React.Fragment>
+              {children}
```

Comment:
future: consider restructure to include this as part of `children` feels a little silly to have a separate aspect, but give the use of the callback, understandable